src/Compilers/VisualBasic/VbcCore/vbc.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+THISDIR=$(dirname $0)
+
+chmod +x $THISDIR/CoreRun 2>/dev/null
+$THISDIR/CoreRun $THISDIR/vbc.exe $@
```

Comment:
While forwarding any argument that has spaces in it, using `$@` will break the argument up.

`csc "C:\Program Files\foo.cs"` becomes `csc "C:\Program" "Files\foo.cs"` once forwarded.  And that's *with* the user manually inserting quotes.